src/frontend/src/lib/components/onramper/OnramperWidget.svelte

```diff
@@ -71,4 +71,5 @@
 	height="660px"
 	width="100%"
 	allow="accelerometer; autoplay; camera; gyroscope; payment; microphone"
+	sandbox="allow-forms allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts"
```

Comment:
In addition to sandbox itself, is that the list of privileges prodsec recommended?

Reply:
They said the ones that are needed for functioning, and those seems like the ones they need to me. I tagged @tmu0 in this PR too, so that he can review